lib/routes/charge/items/item_page.dart

```diff
@@ -14,29 +14,53 @@ import 'package:breez/widgets/single_button_bottom_bar.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-class CreateItemPage extends StatefulWidget {
+import '../currency_wrapper.dart';
+
+class ItemPage extends StatefulWidget {
+  final Item item;
+  final AccountModel accountModel;
   final PosCatalogBloc _posCatalogBloc;
 
-  CreateItemPage(this._posCatalogBloc);
+  ItemPage(this._posCatalogBloc, {this.item, this.accountModel});
```

Comment:
I see we need the accountModel in both new/edit flows. If that is so, it shouldn't be optional.

Reply:
AccountModel is removed and it's being listened once on didChangeDependencies.
[680c91](https://github.com/breez/breezmobile/pull/296/commits/680c91ca295661f5e9c2ba5a5572a18568a3c193)